src/components/Stepper/index.tsx

```diff
@@ -119,14 +119,16 @@ const StyledContainer = styled(Box, {
   shouldForwardProp: prop => !['size'].includes(prop.toString()),
 })<{ disabled: boolean; size: ContainerSizesType }>`
   background-color: ${({ theme, disabled }) =>
-    disabled ? theme.colorsDeprecated.gray100 : theme.colorsDeprecated.white};
+    disabled
+      ? theme.colors.neutral.backgroundDisabled
+      : theme.colors.neutral.backgroundWeak};
   display: flex;
   flex-direction: row;
   align-items: center;
   align-self: stretch;
   font-weight: 500;
   height: ${({ size }) => containerSizes[size]}px;
-  border: 1px solid ${({ theme }) => theme.colorsDeprecated.gray300};
+  border: 1px solid ${({ theme }) => theme.colors.neutral.border};
```

Comment:
```suggestion
  border: 1px solid ${({ theme }) => theme.colors.neutral.borderWeak};
```

Same here it is a weak context as background is light :)